scripts/download-sample-sheets

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+HOST=$1
```

Comment:
Another option instead of reading from .env variables is `HOST="${1:=http://localhost:3100}"` so we fallback to local if one is not provided. Not sure if that is better or not, will leave it up to you